app/controllers/admin/delete_work_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller for deleting a work as an administrator
+  class DeleteWorkController < Admin::ApplicationController
+    def new
+      authorize!
+
+      @delete_form = Admin::DeleteForm.new
```

Comment:
Using a form isn't needed for this function.

Reply:
I'll work through this, but what I found was that not having a form screws up how we render form fields and I didn't want to have a form that goes away from our establish patterns.

Reply:
That was the special sauce I was missing!